dist/ng-wig.js

```diff
@@ -13,10 +13,11 @@ angular.module('ngWig').component('ngWigPlugin', {
     plugin: '<',
     execCommand: '=',
     editMode: '=',
-    disabled: '='
+    disabled: '=',
+    options: '<'
   },
   controller: ["$scope", "$element", "$compile", function ($scope, $element, $compile) {
-    $element.replaceWith($compile('<' + this.plugin.pluginName + ' ' + 'plugin=' + '"$ctrl.plugin"' + 'exec-command=' + '"$ctrl.execCommand"' + 'edit-mode=' + '"$ctrl.editMode"' + 'disabled=' + '"$ctrl.disabled"' + '/>')($scope));
+    $element.replaceWith($compile('<' + this.plugin.pluginName + ' ' + 'plugin=' + '"$ctrl.plugin"' + 'exec-command=' + '"$ctrl.execCommand"' + 'edit-mode=' + '"$ctrl.editMode"' + 'disabled=' + '"$ctrl.disabled"' + 'options=' + '"$ctrl.options"' + '/>')($scope));
```

Comment:
I might be overstepping the scope of the ngWig editor here, in which case I can maintain it as a separate repository for our purposes. I'll tell you the use-case where I needed options.

We needed an editor for building mail models, in which you can add placeholders (columns from a table). For this I added a new button and created a custom plugin. But the plugin must be told what table to select columns from, and I found no way to pass this information from outside ngWig to the  plugin.